examples/sink-get-200.json

```diff
@@ -0,0 +1,177 @@
+{
+  "id": "171d5c87-7fff-4c53-9383-46503eb1c23e",
+  "version": "1441704616:592733242",
+  "label": "Display 1",
+  "description": "Display placed on the table in the kitchen",
+  "tags": [],
+  "edid": {
+    "manufacturer": "SAM Model b43",
+    "manufacture_week": 19,
+    "manufacture_year": 2018,
+    "version": "1441704616:592733242",
+    "screen_size": {
+      "width": 53,
+      "height": 30
+    },
+    "gamma": 2.2,
+    "display_type": "RGB 4:4:4 + YUV 4:4:4",
```

Comment:
Perhaps something more machine parsable? Array of enums?

Reply:
You're extremely right. I will change it.